### PR TITLE
Swapping Entire Nodes Without Direct Value Swap

### PR DESCRIPTION
**Pull request by Saireddy1369**
1. In most solutions, after locating the nodes at a k distance from the start and end, the values of the nodes are directly swapped, which is a basic approach.
2. In my solution, instead of swapping values, the entire nodes are swapped by carefully adjusting the pointers while maintaining the previous and next nodes of the target nodes.

In simpler terms, instead of swapping the data values -> my solution swaps the memory location of the nodes.

The important function is swapNodes(ListNode head, int k), while the other functions are provided to support  the execution of test cases.

### CODE REVIEW

**Review by aman247av — CHANGES_REQUESTED** [review]
please change file naming... it should be LT_Swapping_Nodes_in_a_Linked_List.java

**Comment by Saireddy1369** [comment]
done changed file name
